example/offboard_velocity/offboard_velocity.cpp

```diff
@@ -99,31 +98,24 @@ bool offb_ctrl_ned(Device &device)
     // NOTE: Use sleep_for() after each velocity-ned command to closely monitor their behaviour.
 
     offboard_log(offb_mode,  " Turn clock-wise 270 deg");
-    for (unsigned i = 0; i < 400; ++i) {
-        device.offboard().set_velocity_ned({0.0f, 0.0f, 0.0f, 270.0f});
-        sleep_for(milliseconds(10));
-    }
+    device.offboard().set_velocity_ned({0.0f, 0.0f, 0.0f, 270.0f});
+    sleep_for(seconds(2));
+
     offboard_log(offb_mode, " Done");
 
     offboard_log(offb_mode, " Go UP 2 m/s, Turn clock-wise 180 deg");
```

Comment:
@julianoes When this command executes the vehicle should turn clockwise from 270 through to 0 and back round to 180 - because the yaw value is positive. What I'm seeing in SITL is that it goes anticlockwise back to 180 irrespective of the sign on the yaw value. So either a bug in our docs, or dronecore or PX4.

Reply:
It just goes the quickest way to the new setpoint. I'll get rid of the "(anti-)clock-wise" indications.